tests/tests.py

```diff
@@ -198,7 +198,134 @@ def test_elb(self):
         config = ConfigParser(
             ProjectConfig(
                 'sample-project.yaml',
-                'dev').config)
+                'dev').config,'my-stack-name')
+        self.assertEquals(known, config.elb())
+
+    def test_elb_missing_cert(self):
+        from testfixtures import compare
+
+        self.maxDiff = None
+        project_config = ProjectConfig('sample-project.yaml', 'dev')
+        # Ugh. Fixtures please?
+        project_config.config.pop('ssl')
+        project_config.config['elb'] = [{
+            'name': 'dev_docker-registry.service',
+            'hosted_zone': 'kyrtest.foo.bar.',
+            'certificate_name': 'my-cert',
+            'scheme': 'internet-facing',
+            'listeners': [
+                { 'LoadBalancerPort': 80,
+                  'InstancePort': 80,
+                  'Protocol': 'TCP'
+                },
+                { 'LoadBalancerPort': 443,
+                  'InstancePort': 443,
+                  'Protocol': 'HTTPS'
+                },
+            ],
+        }]
+        config = ConfigParser(project_config.config,'my-stack-name')
+        self.assertRaises(errors.CfnConfigError)
```

Comment:
You aren't actually testing anything here: when called like this it is returning an Context Manager: https://docs.python.org/2.7/library/unittest.html#unittest.TestCase.assertRaises

```python
with self.assertRaises(errors.CfnConfigError):
  config.ssl()
```

is what you want